internal/gatewayapi/testdata/httproute-with-header-filter-duplicate-add-multiple-filters.out.yaml

```diff
@@ -71,10 +71,6 @@ httpRoutes:
       conditions:
       - type: Accepted
         status: "True"
```

Comment:
I'd think this should be `status: "False"` if there's a reason of UnsupportedValue?

Reply:
@skriss commit e14991a updates the PR to only set the "UnsupportedValue" reason when `Accepted=false`.